returnn/util/file_cache.py

```diff
@@ -64,14 +66,19 @@ def __init__(
         :param cleanup_files_wanted_older_than_days: if cleanup_disk_usage_wanted_free_ratio not reached,
             cleanup files older than this.
         :param cleanup_disk_usage_wanted_free_ratio: try to free at least this ratio of disk space.
+        :param num_retries: how many times to try caching a file before giving up
         """
+
+        assert num_tries > 0
```

Comment:
This should be moved below to where it is assigned. Also without the empty lines.